conf/VERSION.properties

```diff
@@ -1,4 +1,4 @@
 
 ## Defines the JGroups version
 ## This file should be the *only place* to change when changing the version number
-jgroups.version=3.6.4.Final
\ No newline at end of file
+jgroups.version=4.0.0-SNAPSHOT
```

Comment:
There are no wildfly rules. I am talking about jboss rules https://developer.jboss.org/wiki/JBossProjectVersioning and maven convetions, both of which would be nice to follow.